tests/devices/i20/test_gas_injector.py

```diff
@@ -0,0 +1,221 @@
+from unittest.mock import AsyncMock, call, patch
+
+import pytest
+
+from dodal.devices.i20.gas_injector import (
+    GasInjector,
+    GasToInject,
+    IonChamberToFill,
+    PressureMode,
+    VacuumPumpCommands,
+    ValveCommands,
+)
+
+
+@pytest.fixture
+def injector():
+    # Patch epics_signal_rw and epics_signal_r to return AsyncMocks
+    with (
+        patch(
+            "dodal.devices.i20.gas_injector.epics_signal_rw",
+            side_effect=lambda *a, **k: AsyncMock(),
+        ) as _,
+        patch(
+            "dodal.devices.i20.gas_injector.epics_signal_r",
+            side_effect=lambda *a, **k: AsyncMock(),
+        ),
+    ):
+        yield GasInjector("BLXX:")
```

Comment:
```suggestion
    # Patch epics_signal_rw and epics_signal_r to return AsyncMocks
    with init_devices(mock=True):
        yield GasInjector("BLXX:")
```

Why do you need the signals to be AsyncMocks? Don't you just need them to have Mock backends?